boards/xtensa/esp32/Kconfig.defconfig

```diff
@@ -10,9 +10,31 @@ config BOARD
 config ENTROPY_ESP32_RNG
 	default y if ENTROPY_GENERATOR
 
+config BT_RESERVE_DRAM
```

Comment:
What option is this? It doesn't currently exist in a tree, i.e. it seems you're not just setting a default value for it here but this is the actual definition of it. Since it's esp32-specific I don't think you should use the `BT_*` namespace but use something with ESP32 in the name.

Reply:
Yes, it's missing its definition in this case. I'll fix that and add the prefix if necessary.

Reply:
@jhedberg, where would be the proper place to set ESP32 specific BT configs?

Reply:
I'm not completely sure, but `soc/xtensa/esp32/Kconfig.soc` would be one option, since you're using the symbol in the linker script for this SoC.